packages/snowpack/src/commands/dev.ts

```diff
@@ -191,16 +190,6 @@ export async function command(commandOptions: CommandOptions) {
   }
 
   const messageBus = new EventEmitter();
-  console.log = (...args) => {
-    messageBus.emit(paintEvent.CONSOLE_INFO, {msg: args.join(' ')});
-  };
-  console.warn = (...args) => {
-    messageBus.emit(paintEvent.CONSOLE_WARN, {msg: args.join(' ')});
-  };
-  console.error = (...args) => {
-    messageBus.emit(paintEvent.CONSOLE_ERROR, {msg: args.join(' ')});
-  };
```

Comment:
Now we no longer shim `console.*`, **HOWEVER**, we have to pass everything through `logger.*` to not lose the logs.

Reply:
yea, we may want to keep shimming these. Rollup is still using console.log for its internal warnings, for example.

Reply:
Hm. If we shim `console.log` here, then we’d have to use `process.stdout` in our logger (to prevent an infinite loop). Does that work?

Reply:
Actually, yeah—I’ll replace the shim and it’ll work _because_ we’re calling `logger.on('…')`, which would prevent that ultimate loop. We’ll just have to ensure that the two are used togother.